client/src/Components/Employees/taskPropertyHelper.js

```diff
@@ -0,0 +1,46 @@
+const taskPropertyAssign = opts => {
+  const {
+    employees,
+    tasks,
+    properties
+  } = opts;
+
+  if (tasks && properties) {
```

Comment:
This is another spot where I'd recommend adding a comment about the purpose, and when it's being used (like the fact that you are only calling it into action if the user routes directly to the single employee page).

Reply:
I should definitely add some comments to my code. I think I'm going to start a Trello item to take notes of these things and go back to them later this week.